engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/views/product-repositories.html

```diff
@@ -17,7 +17,7 @@
 
     <button type="button" class="btn btn-default"
             ng-click="openModal()"
-            ng-show="canRemoveRepositories(product)"
+            ng-show="product.$resolved && canRemoveRepositories(product)"
```

Comment:
should we disable these buttons instead of hiding them?

Reply:
I think we disable buttons for repos we *can* remove but haven't checked any yet. We still want to hide the button completely for rhel repos and users without permission. At least, that's my understanding of the way it is currently implemented.

Reply:
ah, i guess yeah these buttons are hidden for those reasons, so its better to have buttons appear after a second or two than disappear after a second or two

Reply:
This change shouldn't affect whether we hide/show/disable/enable any of the buttons. It just waits to show them until we have the proper data to determine what to do.